### DISCUSSION TITLE
SIGGRAPH Birds of a Feather?

### DISCUSSION THREAD

**Discussion by randypacker** [dreamworksanimation/openmoonray#97] [Announcements]
Hello!  We’re considering a Birds of a Feather session for MoonRay at this year’s SIGGRAPH.   A chance to mingle and share information and thoughts in a casual atmosphere with DreamWorks engineers and hopefully other people leading MoonRay in interesting directions in this forum.  Are there any particular topics folks would hope are covered in such a session?  Your feedback, advice and participation is greatly appreciated.  Thanks!

**Comment by randypacker** [comment]
Following up on this, we're confirmed to have a BoF in participation with the Academy Software Foundation's Open Source Days during SIGGRAPH on Monday, August 7th.  Exact timeslot tbd.  If we can, we'll try to arrange a live demo of multi-machine rendering with MoonRay and Arras.  More details to follow.  We hope to chat and talk shop soon!

**Reply by randypacker** [reply]
The schedule has been posted, and our Birds of the Feather will be 5PM on Monday, August 7th. as part of Open Source Days by the Academy Software Foundation.  Please join us!  I think there'll be some interesting stuff to show and discuss.

https://osdbof2023.sched.com/event/09fe70bf7662a62c23e48a1f69ae731c

**Reply by panda1100** [reply]
@randypacker Thank you Randy-san for announcement 🎉 

Speaking of multi-machine with Arras, is that possible to choose what interface (and/or ip address) to be used when registering nodes to the pool (that coordinator manages) using `arras4_node` with `--no-consul` option.

My testing environment has two network interfaces, one is `eth0` the other is `wireguard`. mulit-node rendering works good with `eth0`, but for distributed multi-node rendering, I would like to use `wireguard` interface, but `arras4_node` always choose `eth0` ip address. It looks like `/etc/hosts` and DNS doesn't play any role when `arras4_node` choose what ip address to be used. (@toshi-kato -san) Maybe I should create new thread though..

**Reply by rgwilson68** [reply]
arras4_node sends a list of all interfaces to coordinator, but picks the first one as default (see ArrasNode::fetchInetInterfaces() in arras4_node/cmd/node/ArrasNode.cc). minicoord currently always picks the default. So you could modify arras4_node to allow the default interface to be set as a command line option, or add something to minicoord to pick one of the non-defaults..

**Reply by panda1100** [reply]
Thank you @rgwilson68 Rob-san! The former approach (push approach), add option to arras4_node for setting default interface, makes sense for me. Hope I can work on this soon..
